src/ASM/SAMpatch.h

```diff
@@ -54,6 +61,8 @@ class SAMpatch : public SAM
   //! \a MPMCEQ, \a MMCEQ and \a TTCC.
   virtual bool initConstraintEqs(const std::vector<ASMbase*>& model);
 
+  std::vector<ASMbase*> patches; //!< The spline patches
+
 private:
```

Comment:
hovedgrunnen er at jeg tar ikke vare på SIM-referansen, så den er kun tilgjengelig i DomainDecomposition::setup. jeg trenger tilgang til patchene senere i forbindelse med oppsett av lokale subdomener for ASM-prekondisjonering. se https://github.com/OPM/IFEM/pull/7/files/7bdbbf124bf39a7c802b6418d5926dedcb8e468f..0bb8789bacdece8c4bc6008eb2058cab563bcdc1#diff-89249d74ae2533f6c7c69eae589f2375R99

et alternativ ville vært å ta vare på SIM-referansen. jeg valgte å la SAMpatch ta vare på patchene den sendes i constructoren da det passer inn med rekkefølge på ting og tang og at det virket merkelig at SAMpatch ikke vet av sine patcher. jeg kan fint forandre..